internal/storage/storageutil/object_attrs_test.go

```diff
@@ -0,0 +1,102 @@
+package storageutil
+
+import (
+	"crypto/md5"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/storage"
+	. "github.com/jacobsa/ogletest"
+	storagev1 "google.golang.org/api/storage/v1"
+)
+
+const TestBucketName string = "gcsfuse-default-bucket"
+const TestObjectName string = "gcsfuse/default.txt"
+const Test string = "test"
+
+func TestObjectAttrsToBucketObject(t *testing.T) { RunTests(t) }
+
+type objectAttrsTest struct {
+}
+
+func init() { RegisterTestSuite(&objectAttrsTest{}) }
+
+func (t objectAttrsTest) TestObjectAttrsToBucketObjectMethod() {
+	var attrMd5 []byte
+	Time := time.Now()
+	attrs := storage.ObjectAttrs{
+		Bucket:                  TestBucketName,
+		Name:                    TestObjectName,
+		ContentType:             Test,
+		ContentLanguage:         Test,
+		CacheControl:            Test,
+		EventBasedHold:          true,
+		TemporaryHold:           true,
+		RetentionExpirationTime: Time,
+		ACL:                     nil,
+		PredefinedACL:           Test,
+		Owner:                   Test,
+		Size:                    16,
+		ContentEncoding:         Test,
+		ContentDisposition:      Test,
+		MD5:                     attrMd5,
+		CRC32C:                  0,
+		MediaLink:               Test,
+		Metadata:                nil,
+		Generation:              780,
+		Metageneration:          0,
+		StorageClass:            Test,
+		Created:                 Time,
+		Deleted:                 Time,
+		Updated:                 Time,
+		CustomerKeySHA256:       Test,
+		KMSKeyName:              Test,
+		Prefix:                  Test,
+		Etag:                    Test,
+		CustomTime:              Time,
+	}
+	CustomeTimeExpected := string(attrs.CustomTime.Format(time.RFC3339))
```

Comment:
please fix everywhere. variables names should start with small letter.

Reply:
fixed